### PR TITLE
[FE-6169] Support Shorthand Region Groups

### PR DESCRIPTION
**Pull request by henryfauna**
Ticket(s): FE-6169

## Problem
You must use the region groups that the account api expects as input to the `--database` arg. eg. `--database us-std/my-database`. We do not often expose the name `us-std` to customers and it will likely be more natural to use `--database us/my-database` as that is what is presented in the dashboard.

## Solution
Standardize region groups before they are passed to the account API. This allows the usage of `--database us-std/my-database` or `--database us/my-database`.

## Result
You can use whatever region group reference you prefer.

## Testing
Updated existing tests.

### CODE REVIEW

File: src/lib/fauna-account-client.mjs [hunk 1]
```diff
@@ -204,12 +207,50 @@ export class FaunaAccountClient {
...
+   * // Returns "eu-std/my-database"
+   * standardizeRegion("eu/my-database")
+   *
+   * // Returns "global/my-database"
+   * standardizeRegion("classic/my-database")
+   *
+   * @throws {TypeError} If databasePath is provided but not a string
+   */
+  standardizeRegion(databasePath) {
+    if (!databasePath) return databasePath;
```
Comment (echo-bravo-yahoo) [inline src/lib/fauna-account-client.mjs:238]: what case is this handling & should we prefer to throw instead?

File: src/lib/fauna-account-client.mjs [hunk 2]
```diff
@@ -204,12 +207,50 @@ export class FaunaAccountClient {
...
+    const rest = parts.slice(1).join("/");
+
+    const regionMap = {
+      us: "us-std",
+      eu: "eu-std",
+      classic: "global",
+    };
+
+    const standardRegion = regionMap[region] || region;
+    return `${standardRegion}/${rest}`;
```
Comment (echo-bravo-yahoo) [inline src/lib/fauna-account-client.mjs:254]: can you add a test (general-cli.mjs maybe?) that confirms the regions get mapped?

Comment (henryfauna) [reply src/lib/fauna-account-client.mjs]: Added!

File: src/lib/fauna-account-client.mjs (hunk 1 above)
Comment (henryfauna) [reply src/lib/fauna-account-client.mjs:238]: I don't think so - an undefined or empty path is valid in some cases. For instance on the list database API.

**Review by mwilde345 — APPROVED** [review]

**Review by ecooper — APPROVED** [review]

**Pull request merged**